client/src/components/dataset/DatasetTableV2.jsx

```diff
@@ -77,6 +80,61 @@ function DatasetTable(props) {
     });
   };
 
+  useEffect(() => {
+    const intro = introJs();
+
+    intro.onbeforechange(() => {
+      const currentStepIdx = intro._currentStep;
+      const currentStepDynamic = !!intro._options.steps[currentStepIdx].dynamic;
+
+      if (currentStepDynamic) {
+        const step = intro._options.steps[currentStepIdx];
+        const element = document.querySelector(step.element);
+
+        if (element) {
+          const introItem = intro._introItems[currentStepIdx];
+          introItem.element = element;
+          introItem.position = step.position;
+        }
+      }
+    });
+
+    intro.setOptions({
+      nextLabel: 'Next',
+      prevLabel: 'Back',
+      hidePrev: true,
+      skipLabel: 'Skip',
+      doneLabel: 'Got it',
+      showStepNumbers: false,
+      steps: [
+        {
+          element: '#GroupsList',
+          intro: 'Data groups are based on Flow form question groups and submission metadata',
```

Comment:
Should the strings be in a strings resource file so we can have them translated?